_posts/2012-01-19-format-efficiency-take-2-really-clean-json.html

```diff
@@ -112,30 +112,30 @@ <h2>Approach</h2>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>W/"<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">Apply</span><span style="color: #ff0000">Function</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">RawValue</span>"</span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">Apply </span><span style="color: #e50000">Function</span><span style="color: #0000ff">=</span>"<span style="color: #0000ff">RawValue</span>"</span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>          &lt;</span></span><span><span style="color: #a31515">Path</span><span style="color: #0000ff">&gt;</span>Version<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">Path</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
 <p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>        &lt;</span></span><span><span style="color: #a31515">String</span><span style="color: #0000ff">&gt;</span>"<span style="color: #0000ff">&lt;/</span><span style="color: #a31515">String</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply</span></span><span style="color: #0000ff">&gt;</span></span></p>
-<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>      &lt;/</span></span><span><span style="color: #a31515">Apply </span></span><span style="color: #0000ff">&gt;</span></span></p>
+<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation </span></span><span style="color: #0000ff">&gt;</span></span></p>
```

Comment:
```suggestion
<p><span style="font-family: Lucida Console"><span style="color: #0000ff"><span>    &lt;/</span></span><span><span style="color: #a31515">ValueAnnotation</span></span><span style="color: #0000ff">&gt;</span></span></p>
```
This is a closing `</ValueAnnotation>`. We shouldn't have a space after `ValueAnnotation`